chain/actors/actors.go

```diff
@@ -0,0 +1,332 @@
+package actors
```

Comment:
This file will need to exist until a lotus release containing https://github.com/filecoin-project/lotus/pull/8941 is cut. It ensures the correct actor CID's are returned for actors v8 since their CID's will differ depending on the network: mainnet, calibnet, buttnet, etc. Actors v7-0 remain unchanged ofcourse